src/components/Agreement/Agreement.js

```diff
@@ -13,7 +15,14 @@ const Agreement = React.memo(function Agreement() {
       <Split
         primary={
           <>
-            <Box>Details</Box>
+            <Box>
+              <AgreementHeader title="Aragon Network DAO Agreement" />
+              <AgreementDetails
+                contractAddress="0x5c6620c49f9aecf74bd483054f2d0ace0d375f96"
+                creationDate="2020/07/20"
```

Comment:
Maybe here we can pass the date in seconds like the one that is in the subgraph

Reply:
Definitely agree, how'd you feel if I do this in a separate improvement? I don't want to introduce too many more conflicts with date utils in our outstanding PRs.

Reply:
Sounds good!